native/libp2p_nif/go.sum

```diff
@@ -10,6 +10,8 @@ git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGy
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
there shouldn't be any changes to this file

Reply:
it happened when I run the `make spec-test` actually, I will revert to the old `go.sum` file